src/Scouting.elm

```diff
@@ -0,0 +1,65 @@
+module Scouting exposing (Msg, Model, init, update, main, view )
```

Comment:
Use `elm-format`.
It should be configured to run automatically whenever you save.
Make sure your preferences contain these settings:
![image](https://user-images.githubusercontent.com/25708271/70334301-dc8d0800-184d-11ea-8ff6-34b6c7bc25e6.png)
(open settings with `ctrl-p` `>` `open settings json`)